src/components/Application/Welcome.tsx

```diff
@@ -15,9 +15,11 @@ import { useStore } from './store'
 
 export default function Welcome() {
   const setIsWelcome = useStore((state) => state.setIsWelcome)
+  const theme = useTheme()
+  const height = `calc(100vh - ${theme.spacing(8)})`
 
   return (
-    <Grid container>
+    <Grid container sx={{ height, overflowY: 'auto' }}>
```

Comment:
@shashigharti I changed here to `auto` as the scroll is not needed when it's enough space